src/components/PageHeader/PageHeader.tsx

```diff
@@ -42,10 +42,11 @@ export interface Props {
    */
   headingVariant?:
     | 'error'
-    | 'base'
     | 'brand'
     | 'inherit'
-    | 'neutral'
+    | 'neutralSubtle'
```

Comment:
maybe outside scope of this PR, but could this import the `Variant` type from `Heading`?

Reply:
Is it cool if I clean that up in a separate PR? I don't want to bloat this one with that change.

Reply:
def!